.github/workflows/push-pypi.yml

```diff
@@ -0,0 +1,26 @@
+name: Push monocle to pypi
+
+on:
```

Comment:
which branch does this run on ? should we pass the release branch or tag that would be needed later ?

Reply:
The branch name would need to be passed